DoctorWho.Db/Repositories/EpisodeRepository.cs

```diff
@@ -0,0 +1,134 @@
+﻿using AutoMapper;
+using Dapper;
+using DoctorWho.Db.DatabaseContext;
+using DoctorWho.Db.Entities;
+using DoctorWho.Db.ViewsModels;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using DoctorWho.Web.Models;
+using System;
+using DoctorWho.Common.Models;
+
+namespace DoctorWho.Db.Repositories
+{
+    public class EpisodeRepository
+    {
+        private readonly DoctorWhoCoreDbContext _context;
+        private readonly AuthorRepository _authorRepository;
+        private readonly DoctorRepository _doctorRepository;
+        private readonly string _connectionString;
+        private IMapper _mapper;
+
+        public EpisodeRepository(DoctorWhoCoreDbContext context
+            ,AuthorRepository authorRepository
+            ,DoctorRepository doctorRepository
+            ,IConfiguration configuration
+            ,IMapper mapper)
+        {
+            _context = context;
+            _authorRepository = authorRepository;
+            _doctorRepository = doctorRepository;
+            _connectionString = configuration.GetConnectionString("DBConnectionString");
+            _mapper = mapper;   
+        }
+        public EpisodeDto CreateEpisode(EpisodeDto episodeDto)
+        {
+            var episodeEntity = _mapper.Map<Episode>(episodeDto);
+            _context.Episodes.Add(episodeEntity);
+            _context.SaveChanges();
+
+            var createdEpisodeDto = _mapper.Map<EpisodeDto>(episodeEntity);
+            return createdEpisodeDto;
+
+        }
+        public void UpdateEpisode(int episodeId, string newEpisodeTitle, DateTime newEpisodeDate, int newAuthorId)
+        {
```

Comment:
I know it's not required but for a tip for you, update the parameters to pass object of type EposideDto and mapping to DB model is better